app/dts/bindings/kscan/zmk,kscan-sideband-behaviors.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: |
+  kscan sideband behavior runner. Only basic system behavior should be used,
+  since no keymap processing occurs when using them.
+
+compatible: "zmk,kscan-sideband-behaviors"
+
+include: kscan.yaml
+
+properties:
+  kscan:
+    type: phandle
+    required: true
+
+child-binding:
+  description: "A sideband behavior tied to a row/column pair"
+
+  properties:
+    row:
+      type: int
+      required: true
```

Comment:
Could `row` be made optional (defaulting to 0) to simplify use with the direct GPIO kscan driver (and anything else that creates a flat list instead of a matrix)?

Reply:
Done. Good call.